interfaces/transaction.ts

```diff
@@ -1,17 +1,20 @@
 export enum TRANSACTION_TYPES {
   SET_DATA = 'SET_DATA',
+  SET_DATA_BATCH = 'SET_DATA_BATCH',
   EXECUTE = 'EXECUTE',
   TRANSFER_OWNERSHIP = 'TRANSFER_OWNERSHIP',
 }
 
 export const TRANSACTION_SELECTORS = {
-  SET_DATA: '14a6e293',
+  SET_DATA: '7f23690c',
```

Comment:
I think it would be better if we search the abi by name and then use the signature to calculate the hash. That way things will automatically update. If a lookup is too much even just adding a comment with the complete function signature would this be more findable.

Reply:
I tried to do this though the transaction selectors are not available in the exported constants by name, only by transaction selector

Reply:
Yea that's what we use in the extension but these specific ones might be ok but worth a comment like 
```js
const var = "..." // =kekkak256("...").slice(2, 10)
```
That way they'll show up when searching for `setData(bytes32,bytes)` for example. Otherwise they are hard to track in the future